src/hooks/useClickOutside/useClickOutside.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useRef } from 'react';
+import { fireEvent, render } from '@testing-library/react';
```

Comment:
`import { render, screen, fireEvent } from '~lib/test-utils';`

lib/test-utils basically re-exports testing-library with a customRender function (and could be used eventually to export more helpers/providers etc).